install/kubernetes/helm/istio/charts/pilot/templates/deployment.yaml

```diff
@@ -67,8 +67,6 @@ spec:
               fieldRef:
                 apiVersion: v1
                 fieldPath: metadata.namespace
-          - name: PILOT_THROTTLE
-            value: "500"
```

Comment:
@costinm ^^^ what was this env var for? I think we have way too may env vars in Pilot now, peppered all over the code. We need to consolidate them before we forget what is what. Or remove them.